src/components/AddBook.jsx

```diff
@@ -21,13 +24,3 @@ export default function AddBook({ bookList, setBookList }) {
     </form>
```

Comment:
- [ ] Hi @jaimevillegas, I must admit the fact that you are doing well in this project. I'll kindly once more request you to check the requirements of this project.

![Screenshot from 2023-04-04 18-37-00](https://user-images.githubusercontent.com/63087491/229873005-77f4b7a2-8a19-4051-ab92-fb3c921d0fcf.png)

- Instead of using an event handler on the form itself let's kindly do it on the < Button > as requested, to add the book to the books(value) array(with attributes title and author), For this specific, I would kindly suggest using onClick event handler.

Reply:
Hello @FrederickMih!

Thank you for the suggestion. I worked on solving that, and know it meets the project requirements.